modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Cursor.scala

```diff
@@ -96,6 +96,11 @@ sealed abstract class Cursor(private val lastCursor: Cursor, private val lastOp:
     */
   def as[A](implicit A: Decoder[A]): Decoder.Result[A] =
     A(this)
+
+  /**
+    * Returns the Graph associated with this cursor.
+    */
+  def graph: Graph = g
```

Comment:
why is it here, instead of directly in the constructor as:
```scala
sealed abstract class Cursor(private val lastCursor: Cursor, private val lastOp: CursorOp, val g: Graph)
```

Reply:
hides the graph reference in the cursor...